spec/features/post_and_view_messages_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/message'
+require './lib/user'
+require 'capybara/rspec'
+require './spec/web_helper'
+
+
+feature '1. Post Peep to Chitter' do
+  scenario 'Post new message and display it beneath' do
+    signup_steps
+    signin_steps
+    fill_in('content', :with => 'Peep peep')
+    click_button('Send')
+    expect(page).to have_content('Peep peep')
+  end
+end
+
+feature '2. View all Peeps' do
+  scenario 'Shows all peeps in reverse chronological order' do
+    signin_steps
+    fill_in('content', :with => 'Peep1')
+    click_button('Send')
+    fill_in('content', :with => 'Peep2')
+    click_button('Send')
+    fill_in('content', :with => 'Peep3')
+    click_button('Send')
+    expect(page).to have_content('Peep1')
+    expect(page).to have_content('Peep2')
+    expect(page).to have_content('Peep3')
+  end
+end
```

Comment:
This is good for testing for the content but less so for the order the content appears in - I'll send you the code I used to test for this (originally courtesy of @jaywayawyaj)